KEYCHAIN.md

```diff
@@ -1,23 +1,21 @@
-# The Keychain
+# ![](https://img.shields.io/badge/status-wip-orange.svg?style=flat-square) The Keychain
 
-Authors: [Juan Benet](github.com/jbenet)
-
-Reviewers:
+**Maintainer(s)**:
+- [Juan Benet](github.com/jbenet)
```

Comment:
Ack.

I've mixed feelings with the value of having an Author. In the world of open collaboration and with everything being version controlled, Authorship gets tracked by the commit log. Unless you see a special responsibility that the original author would have (for ever, given that the original author never changes) that the maintainer could not take.

As a note, in the JS modules, we removed the author field in favor of the maintainer. It also serves to recognize the awesome contributor that is taking the responsibility.

Reply:
I think there is a difference between code and a spec in this case. An example that comes to mind is IETF RFCs, which aren't maintained but updates that supersede old versions are created and it is the authors that are listed. I personally think knowing who authored the original document is important to state even if they are no longer maintaining it.

Reply:
Valuable points. I still have mixed feelings though, specially because there are "specs" that are no more than the placeholder or "specs" that are other docs glued together.

Having an author field also begs to ask: What happens when we have multiple Authors but uneven distribution of the work?

In the end, will this ever be used? Again, we have version control. We have line by line authorship anyway :)